modules/tinymce/src/core/test/ts/browser/undo/UndoKeyboardShortcutsTest.ts

```diff
@@ -0,0 +1,41 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyContentActions } from '@ephox/mcagar';
+import { PlatformDetection } from '@ephox/sand';
+import { TinyHooks } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.core.undo.UndoKeyboardShortcutTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    base_url: '/project/tinymce/js/tinymce'
+  }, [], true);
+  const platform = PlatformDetection.detect();
```

Comment:
```suggestion
  const platform = PlatformDetection.detect();

  const hook = TinyHooks.bddSetupLight<Editor>({
    base_url: '/project/tinymce/js/tinymce'
  }, [], true);
```
Variables should be defined before hooks and ideally be separateded.